### ISSUE TITLE
Attach list of tags on a session

### ISSUE THREAD

**Issue by GerardPaligot** [GerardPaligot/Confily#112]
* In addition to the category of a session, we should be able to attach a list of tags
* In the filter screen, we should be able to filter according to the tags
* Create these tags from Conference Hall and OpenPlanner
